src/table/settings/difficulty_settings.ini

```diff
@@ -92,6 +92,13 @@ strhelp  = STR_CONFIG_SETTING_INDUSTRY_DENSITY_HELPTEXT
 strval   = STR_FUNDING_ONLY
 cat      = SC_BASIC
 
+[SDT_VAR]
```

Comment:
This is meant for what is stored in the PATS chunk according to the document, so this does not quite fit here. I would suggest world_settings.ini.
And I wonder whether it needs to be saved just like `custom_town_number` is, as there is a path from an industry GUI to the access of `customer_industry_number` (via `GenerateIndustries`, which is not called only from initial game generation)

Reply:
My reason for saving in the config file is that this allows a player to store the ideal industry number for their favorite map size, instead of having to remember it. And it's consistent with `custom_town_number`. I could be convinced otherwise. 🙂

Reply:
The whole reason I asked was because I noticed a savegame version in `world_settings.ini` for `customer_town_number`: https://github.com/OpenTTD/OpenTTD/blob/master/src/table/settings/world_settings.ini#L268
Having it saved in the savegame does not exclude it from being saved in the configuration file.
When trying the preview I noticed that the intro game just worked, so a minimum version is not needed anymore since the savegame chunk rewrite. Testing locally also showed that it is actually saved in the savegame already with this change and slightly older versions of OpenTTD are able to load those savegames. So a savegame bump is not needed (anymore).